estate/models/estate_property_tag.py

```diff
@@ -0,0 +1,15 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
```

Comment:
For information, this header is only required for modules inside Odoo/addons or enterprise/. Depending on the team you work for, you may not use it.